cmd/api/src/api/v2/saved_queries_test.go

```diff
@@ -42,7 +45,67 @@ import (
 	"go.uber.org/mock/gomock"
 )
 
+// // Matrix testing experiment
+// Pro:  Unify test logic - later maintenance requires only one test modification, or splitting off new tests when necessary
+// Con:  Poor IDE integration, added complexity, test writer must determine when tests should and shouldn't be consolidated
```

Comment:
We can likely remove this comment now that we've used the matrix testing in a proper test. Ideally we'd probably rewrite all the `ListSavedQueries` test variants into this matrix, but in the interests of time and wanting to move on to more important test writing work, I'm not going to recommend it at this time.